src/ontology/mondo-edit.obo

```diff
@@ -422565,6 +422587,86 @@ is_a: MONDO:0002558 {source="Orphanet:590539"} ! melanotic neurilemmoma
 is_a: MONDO:0016752 {source="Orphanet:590539"} ! benign peripheral nerve sheath tumor
 is_a: MONDO:0018078 {source="Orphanet:590539"} ! soft tissue sarcoma
 
+[Term]
+id: MONDO:0035349
+name: localized dystrophic epidermolysis bullosa
+def: "A localized form of dystrophic epidermolysis bullosa characterized by blisters confined primarily to the hands and feet (acral form) or to the pretibial region (pretibial form). Nail dystrophy or loss is common and may be an isolated finding (nail only form). This disease can be inherited via autosomal dominant or autosomal recessive inheritance." [Orphanet:595356]
+subset: ordo_disease
+synonym: "localized DEB" EXACT [Orphanet:595356]
+xref: Orphanet:595356 {source="MONDO:equivalentTo"}
+is_a: MONDO:0017608 {source="Orphanet:595356"} ! dystrophic epidermolysis bullosa
+
+[Term]
+id: MONDO:0035354
+name: IgG4-related systemic disease
```

Comment:
what is our strategy for ensuring complete classification?

consider obsoleting on merge

Reply:
I created a ticket here: https://github.com/monarch-initiative/mondo/issues/3539